tests/lint/flake8.ini

```diff
@@ -0,0 +1,35 @@
+[flake8]
+
+# Plugins
+use-flake8-tabs = True
+use-pycodestyle-indent = True
+
+# Reporting
+statistics = True
+doctests = True
+show-source = True
+
+# Options
+max-complexity = 15
+max-line-length = 110
+hang-closing = True
+
+ignore =
+	W191, # indentation contains tabs
+	E126, # continuation line over-indented for hanging indent
+	E133, # closing bracket is missing indentation
+	W503, # line break before binary operator. As opposed to W504 (line break after binary operator) which we want to check for.
+
+builtins = # inform flake8 about functions we consider built-in.
+	_, # translation lookup
+	pgettext, # translation lookup
+
+exclude = # don't bother looking in the following subdirectories / files.
+	.git,
+	__pycache__,
+	.tox,
```

Comment:
Do we have this dir?

Reply:
Possibly not, but since there is no point looking in these directories why would we let it?